### DISCUSSION TITLE
Install Augustus in Caesar 3

### DISCUSSION THREAD

**Discussion by gianlore** [Keriew/augustus#438] [General]
I put the contents of the augustus-3.0.1.80-2229b53b-windows file, in the root directory of Caesar 3 (Steam)
Upon departure I receive this message:
![image](https://user-images.githubusercontent.com/72648521/121822482-95cc6980-cc9f-11eb-9309-ac4b419b3570.png)

What can I do?

**Comment by crudelios** [comment]
Get the assets here:  https://augustus.josecadete.net/download/latest/development/assets
Then unzip the file and copy the assets folder to the folder where you have the augustus executable.

**Comment by gianlore** [comment]
I thank you infinitely, I had come to a standstill !!!
